src/style.css

```
@@ -1 +1,73 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+}
+
+.nav__list,
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  padding: 0;
+  flex-grow: 1;
+}
+
+.nav__item {
+  display: flex;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-family: Roboto, serif;
+  font-weight: 500;
+  margin: 0 10px;
+  list-style-type: none;
+  line-height: 60px;
+  text-align: center;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  line-height: 60px;
+  align-items: center;
+  justify-content: center;
```

```suggestion
  align-items: center;
  justify-content: center;
```
Ці дві властивості не працюють без display: flex